### ISSUE TITLE
Zentangles with Overlapping model in WFC

### ISSUE THREAD

**Issue by schrum2** [schrum2/MM-NEAT#329]
Alice, Anna, and Sarah only applied the simple tiled model for WFC in Picbreeder. They left as future work the possibility of using the overlapping model, which I think has a lot more promise. It would probably look a lot cooler too.

**Comment by schrum2** [comment]
I haven't made any changes to the MM-NEAT code for this issue, but I went back to the original repo:
https://github.com/trygvebw/JavaWaveFunctionCollapse
I applied the overlapping model to some tiles created by Picbreeder. The problem isn't that it is hard to do, but rather that the results look like crap. At least ... that's what I'm getting so far. There are other configuration options I need to fiddle with.

Also, all of my successful tests have been with small, simple, black and white images. When the image is too complex, or has color (and is therefore complex) the code takes to long to run and I terminate it early.
